tutorial/vector-similarity-search-open-ai/README.md

```diff
@@ -0,0 +1,11 @@
+# Tutorial: Conduct vector similarity search on Azure OpenAI embeddings using Azure Cache for Redis
+
+## Background
+This repo contains a [juypter notebook](https://jupyter.org/) file with the code used in the [Conduct vector similiarity search on Azure OpenAI embeddings using Azure Cache for Redis](https://aka.ms/redisvsstutorial) tutorial located in the Azure Cache for Redis documentation. Use this example to follow along with the tutorial or as a basis for your own learning. 
+
+## Other Useful Links
+- [More information on Azure Cache for Redis](https://learn.microsoft.com/en-us/azure/azure-cache-for-redis/cache-overview)
```

Comment:
nit: it's good practice to remove "en-us/" from docs links so the doc can be displayed in the language specified by users' settings

Reply:
Oops, good catch!